### PR TITLE
demo login added

### PR DESCRIPTION
**Pull request by bhavy3103**

### CODE REVIEW

File: frontend/src/pages/Login.jsx [hunk 1]
```diff
@@ -42,18 +42,32 @@ const Login = () => {
...
   };
 
-  useEffect(() => {
-    // Close toast when isLoggedIn changes
-    if (authCtx.isLoggedIn) {
-      const closeToast = () => toast.dismiss();
-      setTimeout(closeToast, 3000); // Adjust the timeout as needed
+  const demoLoginHandler = async () => {
+    setUsername('bhavy@gmail.com');
+    setPassword('123');
```
Comment (abhijivani3001) [inline frontend/src/pages/Login.jsx:47]: this state updation is unnecessary

Comment (bhavy3103) [reply frontend/src/pages/Login.jsx:47]: okay removed it

Comment (bhavy3103) [reply frontend/src/pages/Login.jsx:47]: can you say why it is not?
context ma store that chhe ?

**Review by abhijivani3001 — APPROVED** [review]
done

**Pull request merged**